generator/mini_roundabout_processor.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include "generator/feature_builder.hpp"
+#include "generator/osm_element.hpp"
+
+#include "base/math.hpp"
+#include "coding/point_coding.hpp"
+#include "geometry/latlon.hpp"
+#include "geometry/mercator.hpp"
+#include "geometry/point2d.hpp"
+#include "indexer/classificator.hpp"
+
+#include <vector>
+
+using namespace feature;
+
+namespace generator
+{
+/// \brief Calculate Euclidean distance between 2 points on plane.
+double DistanceOnPlain(m2::PointD const & a, m2::PointD const & b);
+
+/// \brief Partition a directed line segment into a ratio |segm_point, target|/|r|.
+/// \return The point that is |segm_point, target| distance from segm_point and |r| distance from
+/// target.
+m2::PointD TrimSegment(m2::PointD const & segm_point, m2::PointD const & target, double r);
+
+/// \brief Transform a point to the hexagon with center in |center|.
+/// \return vector of hexagon vertices
+std::vector<m2::PointD> PointToCircle(m2::PointD const & center, double r);
+
+/// \brief Insert point (which lies on circle) between 2 points already present in the circle.
```

Comment:
Probably it'll be better writing like `/// \brief Inserts point ...` or  `/// \brief Adds ways with ...`? `/// \brief Insert point ...` or  `/// \brief Add ways with ...` sounds like an imperative. What do you think?

Reply:
Agreed.